src/App.test.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-undef */
```

Comment:
Write a test case for this flow:
1. Click on a date.
2. Add a memo.
3. Close it.
4. Again click on the same date to check if the memo persists.